libnmstate/netapplier.py

```diff
@@ -200,10 +209,11 @@ def _list_new_interfaces(desired_state, current_state):
 
 def _verify_change(desired_state):
     current_state = state.State(netinfo.show())
-    desired_state.verify_interfaces(current_state)
-    desired_state.verify_routes(current_state)
-    desired_state.verify_dns(current_state)
-    desired_state.verify_route_rule(current_state)
+    verifiable_desired_state = copy.deepcopy(desired_state)
```

Comment:
Is there a problem if when not using `copy.deepcopy`?

If no, keep it mutable is OK for me.

Reply:
Yes, if we do not use `copy.deepcopy` we could find some issues when retrying the verification.